packages/components/locales/en-GB.json

```diff
@@ -35,7 +35,7 @@
   },
   "filterBar.addFiltersMenu.buttonLabel" : {
     "description" : "Menu button label to show additional available filter options",
-    "message" : "Add filters"
+    "message" : "Add Filters"
```

Comment:
Interesting, this is actually counter to [the update we made](https://cultureamp.slack.com/archives/CETLTFG9G/p1735000032401379). We might have to go into smartling and reject this translation